src/utils/helpers/browser/browser.js

```diff
@@ -11,6 +10,13 @@ import Form from './../../../components/form';
 *
 */
 const Browser = {
+  isDomAvailable: () => {
+    return (!!(
+      typeof Browser.getWindow() !== 'undefined' &&
```

Comment:
could you store the result of the function:

```js
const _window = Browser.getWindow();
return !!(
  typeof _window !== 'undefined' &&
  _window.document &&
  _window.document.createElement
);
```

Reply:
Actually the problem with storing it is that it is a permanent value that doesn't reflect the real state of the environment. Also for simple testing reason where we are 'interfering' with the env it to make this function send false, we need thsi value to reflect the new state.

Reply:
I just meant within the scope of the function, so it's not recalling the same function 3 times at once.